webapp-client/src/component/ContentPane.js

```diff
@@ -51,6 +52,23 @@ export default class ContentPane extends React.Component {
     }
   }
 
+  async deleteItem(item) {
+    const url = process.env.REACT_APP_API_URL + '/item/deleteItem/';
+    try {
+      const response = await fetch(url, {
+        method: 'DELETE',
+        headers: new Headers({ 'content-type': 'application/json' }),
+        body: JSON.stringify({ itemId: item._id }),
+      });
+      if (response.status !== 200) {
+        throw new Error(response.statusMessage);
+      }
+      this.setItems(this.props);
+    } catch (error) {
+      this.showErrorMessage();
```

Comment:
where is the render error for delete items

Reply:
I see what you mean, I added state to disambiguate the errors!

Reply:
If it ends up being useful to have multiple error messages in the same component, I can add another method to errorify